.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  # Every Tuesday at 1:20pm UTC
+  schedule:
+    - cron: '20 13 * * 2'
+
+jobs:
+  ci:
+    name: CI
+    # Only run cron on the silverstripe account
+    if: (github.event_name == 'schedule' && startsWith(github.repository, 'silverstripe/')) || (github.event_name != 'schedule')
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    with:
+      # Turn phpcoverage off because it causes a segfault
+      phpcoverage_force_off: true
```

Comment:
Do you have a link to a run that failed for a segfault? I don't see one in creative-commoners?

Reply:
strange. might have been on different reop. have removed